src/structures/game_session.ts

```diff
@@ -307,6 +321,19 @@ export default class GameSession extends Session {
         }
 
         const round = this.round;
+        this.stopHiddenUpdateTimer();
```

Comment:
In another file, u wrap this in
        if (this.gameType === GameType.HIDDEN) {